magicctapipe/scripts/lst1_magic/semi_automatic_scripts/config_general.yaml

```diff
@@ -0,0 +1,40 @@
+mc_tel_ids:
```

Comment:
mc_tel_ids are also in resources/config.yaml

also we could config_general.yaml to config_autoMCP.yaml or something like this, because "general" sounds more like something that is used for all the scripts not only the semi-automatic ones

Reply:
the config.yaml gets installed when we install the pipeline: so, if we want to change the tels:id it is much easier if we use the config_general (I agree, I will change the name) because we don't need a new 'pip install' to actually change the ids.
If you don't use the automatic scripts you have to copy somewhere the config.yaml, modify it and pass the modified version to the scripts; if you use the automatic ones and you don't want to change cleaning.... you can just modify the config general and don't care about what is installed in the pipeline

Reply:
modified config file you can pass to the config_auto_MCP:(general => base_config_file)
and if you want to modify the telescopes you need to either way extract and modify one of the config files. 
The main problem is that if we have mc_tel_id in both configs it is very confusing which one is really used where

Reply:
We rarely modify the IDs, just in case we have to run special analyses, so i think that we can try to remove the IDs from the config_auto_MCP and check if it is user friendly. As above, new PR :)   (hopefully today)